test.sh

```diff
@@ -6,6 +6,8 @@ set -eux
 HOST=http://localhost:8420
 SK=sk_test_12345
 
+LIMIT=10000
+
```

Comment:
There is no need to define a global variable here, please set it where it's used.